### DISCUSSION TITLE
빌드잇 오브젝트 수정사용

### DISCUSSION THREAD

**Discussion by yj098** [naverz/zepeto-studio-kor#890] [5. Q&A]
안녕하세요 유니티로 월드를 제작할려고 하는데

빌드잇의 한옥 모델에서 용마루와 내림마루 부분을 수정해서 사용하고 싶은데
만약에 이렇게 사용하면 저작권에 위반 되나요?? ㅠ
유니티에서 제페토의 빌드잇 모델과 직접 만든 모델을 섞어 사용해도 괜찮나요??

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요. 질문 주신 내용 모두 사용해도 괜찮습니다.
[ZEPETO 활용 콘텐츠(UGC) 이용 가이드라인](https://support.zepeto.me/hc/ko/articles/4414701429145-ZEPETO-%ED%99%9C%EC%9A%A9-%EC%BD%98%ED%85%90%EC%B8%A0-UGC-%EC%9D%B4%EC%9A%A9-%EA%B0%80%EC%9D%B4%EB%93%9C%EB%9D%BC%EC%9D%B8) 참고 부탁 드립니다.
감사합니다.

**Reply by yj098** [reply]
답변 감사드립니다 ㅎㅎ :) :) 👍
